content/md/first-steps/manage-your-attributes.md

```diff
@@ -7,77 +7,147 @@ ee-only: false
-    - **Locale specific**: if it must apply to specific locales not all of them
+    - A **unique value** (Yes/No): if the value for this attribute is unique (field available or required depending on the attribute type)
+    - A **value per channel** (Yes/No): Will your product attribute values differ on the channels?
+    - A **value per locale** (Yes/No): Will your product attribute values differ on the locales?
+    - **Locale specific**: to enable this attribute only to specific locales and not all of them
     - **Usable in grid**: whether or not the attribute can be displayed as a column or used as a filter in the product grid or in the proposal grid (EE only - since 2.2)
-    - **Read only**: only for Enterprise Edition users, when enabled it prevents this attribute from being edited in the UI, it can only be updated through imports, API or rules
-    - **Validation parameters**: depending on the attribute type chosen (see [Validation for the different attribute types](#validation-for-the-different-attribute-types))
-1. Then click on `Save`
+    - **Read only**: when enabled it prevents this attribute from being edited through the UI, it can only be updated through imports, API or rules _(Enterprise Edition only)_
+    - **Validation parameters**: depending on the attribute type chosen (see [Validation for the different attribute types](#validation-for-the-different-attribute-types)).    
+Click on `Save` to create your attribute.
 
-The attribute is now created, if you need to edit an attribute property, please refer to the next section [Edit attribute properties](/articles/manage-your-attributes.html#edit-attribute-properties)
+The attribute is now created, you **can add a translation for each enabled locales** under the `Label translations` tab. If you need to edit some other attribute's property, please refer to the section [Edit attribute properties](/articles/manage-your-attributes.html#edit-attribute-properties)
 
-::: warning
-After being created, it is not possible to change the following properties of an atttribute:
-- its code
-- its type
-- the unique value property
-- the value per channel property
-- the value per locale property
+::: tips
+**Simple** and **multi select** attribute types **do rarely need to be localisable** since all their options can be translated in any enabled locales in the PIM.
 :::
 
 ::: warning
-Please note that the following attribute codes cannot be used in Akeneo PIM:
-
+Please note that the **following attribute codes cannot be used** in Akeneo PIM:    
 `id`, `associationTypes`, `category`, `categories`, `categoryId`, `completeness`, `enabled`, `family`, `groups`, `associations`, `products`, `scope`, `treeId`, `values`, `label`, `parent`
 :::
 
+::: warning
+If you cannot access the Attribute's page or if you cannot add a new attribute, it is probably because you do not have the role permission to. not. Please refer to [Manage your user roles](/articles/build-your-user-roles.html).
```

Comment:
If you cannot access the `Attribute` page

Reply:
Remove the `not.`

Reply:
`Please refer to ... to learn more.`

Reply:
all done